### DISCUSSION TITLE
Can you publish runner-images as Dockers on ghcr.io?

### DISCUSSION THREAD

**Discussion by na-jakobs** [community/community#160662] [Actions]
Can you publish runner-images as Dockers on ghcr.io?

It would be nice to be able to run the same images on self-hosted runners via `container:` 🙏

**Comment by erik-bershel** [comment]
Hey @na-jakobs!
Sorry for the late response. It's near to impossible to build docker image equal to VM images we assemble here. And on the other hand it would be almost useless because of the container size.
